pkg/diagnose/diagnosis/loader.go

```diff
@@ -69,35 +74,93 @@ const (
 	DiagnosisResultMAX              = DiagnosisUnexpectedError
 )
 
+// ToString returns the string representation of the Result
+func (r Result) ToString(colors bool) string {
+	switch colors {
+	case true:
+		switch r {
+		case DiagnosisSuccess:
+			return color.GreenString("PASS")
+		case DiagnosisFail:
+			return color.RedString("FAIL")
+		case DiagnosisWarning:
+			return color.YellowString("WARNING")
+		default:
+			return color.HiRedString("UNEXPECTED ERROR")
+		}
+	default:
+		switch r {
+		case DiagnosisSuccess:
+			return "PASS"
+		case DiagnosisFail:
+			return "FAIL"
+		case DiagnosisWarning:
+			return "WARNING"
+		default:
+			return "UNEXPECTED ERROR"
+		}
+	}
+}
+
 // Diagnosis contains the results of the diagnosis
 type Diagnosis struct {
 	// --------------------------
 	// required fields
 
 	// run-time (pass, fail etc)
-	Result Result
+	Result Result `json:"result"`
 	// static-time (meta typically)
-	Name string
+	Name string `json:"name"`
 	// run-time (actual diagnosis consumable by a user)
-	Diagnosis string
+	Diagnosis string `json:"diagnosis"`
 
 	// --------------------------
 	// optional fields
 
 	// static-time (meta typically)
-	Category string `json:",omitempty"`
+	Category string `json:"category,omitempty"`
 	// static-time (meta typically, description of what being tested)
-	Description string
+	Description string `json:"description,omitempty"`
 	// run-time (what can be done of what docs need to be consulted to address the issue)
-	Remediation string
+	Remediation string `json:"remediation,omitempty"`
 	// run-time
-	RawError string
+	RawError string `json:"rawerror,omitempty"`
+}
+
+// DiagnoseResult contains the results of the diagnose command
+type DiagnoseResult struct {
+	Diagnoses []Diagnoses `json:"diagnoses"`
```

Comment:
We have the `diagnoses` key duplicated. One here and the other for each `Diagnoses`. What about naming this one something different, such `result`, or `runs`. 

What do you think?

Reply:
let's call the root `result` that will fit with the `summary` bellow

Reply:
Finally lets go for runs